app/views/agent/CapturePreferenceView.scala.html

```diff
@@ -17,56 +17,99 @@
 @import config.AppConfig
 @import models.PreferenceModel
 @import forms.PreferenceForm._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 @import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
-@import views.html.templates.errors.ErrorSummary
-@import views.html.templates.inputs.{OptionRevealHelper, Text}
 
-@this(mainTemplate: MainTemplateOriginal,
+@this(mainTemplate: MainTemplate,
       formWithCSRF: FormWithCSRF,
-      errorSummary: ErrorSummary,
-      text: Text,
-      optionRevealHelper: OptionRevealHelper)
+      govukBackLink: GovukBackLink,
+      govukButton : GovukButton,
+      govukErrorSummary: GovukErrorSummary,
+      govukRadios : GovukRadios,
+      govukInput : GovukInput)
 
 @(preferenceForm: Form[PreferenceModel])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@content = {<p><strong class="bold">@messages("capturePref.confirmationEmailRequest")</strong></p>}
-
-@additionalContentAlternative = {
-  <span class="form-field">@messages("capturePref.email")</span>
-  <span id="form-hint" class="form-hint">@messages("capturePref.email.hintAlternative")</span>
+@backLink = {
+    @govukBackLink(BackLink(
+        href = controllers.agent.routes.AgentHubController.show().url,
+        content = Text(messages("base.back"))
+    ))
 }
 
-@backLink = {
-    <a class="link-back" href='@controllers.agent.routes.AgentHubController.show()'>@messages("base.back")</a>
+@yesHtml = {
+    @govukInput(Input(
+        id = "email",
+        name = "email",
+        inputType = "email",
+        label = Label(
+            content = Text(messages("capturePref.email"))
+        ),
+        value = preferenceForm("email-address").value,
+        hint = Some(Hint(
+            content = Text(messages("capturePref.email.hintAlternative"))
+        )),
+        errorMessage = preferenceForm("email").error.map { err =>
+            ErrorMessage(content = Text(messages(err.message)))
+        },
+        classes = "govuk-!-width-one-third",
```

Comment:
Can you make this input box wider? I'm just looking at it and it seems like most email addresses would be too big to display in here without cutting off